plutus-contract/plutus-contract.cabal

```diff
@@ -235,7 +235,19 @@ test-suite plutus-contract-test
   other-modules:
     Spec.Balancing
     Spec.Contract
-    Spec.Contract.TxConstraints
+    Spec.Contract.Ledger.Constraints.MustIncludeDatum
```

Comment:
Would `Spec.Contract.Ledger.Constraints...` be a better module name?

Once we move to `plutus-tx-constraints`, it's not going to be `Ledger.Constraints` anymore, but `Plutus.Tx.Constraints`.

Reply:
Ah right! In the beginning, I thought `Spec.Contract.Ledger.Constraints` would only contain tests for `plutus-ledger-constraints`. However, it was also decided that it would contain tests for `plutus-tx-constraints`, so I think `Spec.Contract.Tx.Constraints` is a better name.